src/main/java/uk/gov/pay/products/config/RestClientConfiguration.java

```diff
@@ -1,12 +1,28 @@
 package uk.gov.pay.products.config;
 
 import io.dropwizard.Configuration;
+import io.dropwizard.util.Duration;
+
+import javax.validation.constraints.NotNull;
 
 public class RestClientConfiguration extends Configuration {
     private String disabledSecureConnection;
 
+    @NotNull
+    private Duration connectTimeout = Duration.seconds(50L);
+
+    @NotNull
+    private Duration readTimeout = Duration.seconds(50L);
+
```

Comment:
I think it's better to configure the `connectTimeout` and `readTimeout` at https://github.com/alphagov/pay-products/blob/master/src/main/resources/config/config.yaml#L74-L75. So for example 
```
jerseyClientConfiguration:
  connectTimeout: 50
  readTimeout: 50
```
If we're not configuring it in the config.yaml then the values might as well be hard coded into the `RestClientFactory` above?

Reply:
The code says that the `connectTimeout` and `readTimeout` configuration items are set to 50 seconds by default if they are not specified in the config.yaml. When an instance of `RestClientConfiguration` class is first created, these variables are initialised to 50s rather than null values. When Dropwizard application starts reading the config file, it then sets these variables to other values (not 50 seconds) specified in the config file.

There is an example of `MessageQueueFactory` class which sets port to 5672 by default under configuration section. Source: https://www.dropwizard.io/en/latest/manual/core.html.

Reply:
Ok, I'll accept this as there's nothing really wrong with hard configuring it in the java class.